src/forms/TogglePill.jsx

```diff
@@ -93,18 +104,17 @@ class TogglePillBase extends React.Component {
 	}
 }
 
-TogglePillBase.protoTypes = {
+TogglePill.protoTypes = {
 	id: PropTypes.string.isRequired,
 	name: PropTypes.string.isRequired,
 	value: PropTypes.string.isRequired,
 	children: PropTypes.node.isRequired,
 	isActive: PropTypes.bool,
 	topic: PropTypes.bool
 };
-TogglePillBase.defaultProps = {
+TogglePill.defaultProps = {
 	isActive: false
 };
 
-const TogglePill = withToggleControl(TogglePillBase);
-export default TogglePill;
+export default withToggleControl(TogglePill);
```

Comment:
the namin gin this file bugs me. I can bring back` TogglePillBase` if need be. Ultimately I think this should be called `ToggleButton `since the classes refer to button.